__tests__/app.test.js

```diff
@@ -50,4 +50,52 @@ describe('GET /api/categories', () => {
             expect(msg).toBe('End-point Not Found');
         });
     });
-});
\ No newline at end of file
+});
+
+describe('GET /api/reviews/:review_id', () => {
+    test('200: Ensures that what is passed IS an Object', () =>{
```

Comment:
Your next test asserts that this is an object, so no need to keep this test afterwards 👍